### PR TITLE
Update linux.md

### PR DESCRIPTION
**Pull request by abernajb**
correcting the path for mac

### CODE REVIEW

File: docs/installation/linux.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ sudo installer -pkg powershell-6.0.0-alpha.9.pkg -target /
 Paths
 =====
 
-* `$PSHOME` is `/opt/microsoft/powershell/6.0.0-alpha.9/`
+* `$PSHOME` is `/usr/local/microsoft/powershell/6.0.0-alpha.9/`
```
Comment (daxian-dbw) [inline docs/installation/linux.md]: `$PSHOME` is `/opt/microsoft/powershell/<version>/` on Ubuntu. So here we probably need to specify on which OS, maybe like `$PSHOME is ... on Ubuntu; ... on OS X; ... on CentOS`?

Comment (abernajb) [inline docs/installation/linux.md]: The section U changed is specific to Mac OS X

Comment (abernajb) [inline docs/installation/linux.md]: Except its not. 

Ok, maybe a * with a link to the OSX Note below?

Comment (daxian-dbw) [inline docs/installation/linux.md]: yes, that sounds right to me. thanks!

**Comment by andschwa** [comment]
I'm going to completely usurp this PR and ask everyone: do you *want* the Linux packages to install to `/usr/local`? Obviously, once distributions' repositories start packaging PowerShell, they'll install to `/usr`. I used `/opt` simply to be consistent with other Microsoft products being distributed as third party packages, and I couldn't ask the community at the time. Now I'm asking 😄

**Comment by jhasse** [comment]
> do you want the Linux packages to install to /usr/local?

No. I expect everything in `/usr/local` not to be handled by the package manager but by myself (e. g. when installing from source).

**Comment by plinnell** [comment]
> I  used /opt simply to be consistent with other Microsoft products being distributed as third party packages, and I couldn't ask the community at the time. Now I'm asking :smile:
> 

For Linux, this is very much an acceptable solution.  /usr/local should never be touched by packaging

File: docs/installation/linux.md (hunk 1 above)
Comment (andschwa) [inline docs/installation/linux.md]: @abernajb were you going to update this PR, or should I close it?

**Comment by andschwa** [comment]
Sounds like using `/opt` for Linux was the right decision, thanks for your input!

**Comment by andschwa** [comment]
Still waiting on author. Closing for now.

**Pull request closed without merging**